test/ui/ResultActionsMenuTest.ts

```diff
@@ -0,0 +1,73 @@
+import * as Mock from '../MockEnvironment';
+import { ResultActionsMenu } from '../../src/ui/ResultActions/ResultActionsMenu';
+import { $$ } from '../../src/utils/Dom';
+
+export function ResultActionsMenuTest() {
+  describe('ResultActionsMenu', () => {
+    let test: Mock.IBasicComponentSetup<ResultActionsMenu>;
+
+    beforeEach(() => {
+      test = Mock.basicResultComponentSetup<ResultActionsMenu>(ResultActionsMenu);
+    });
+
+    afterEach(() => {
+      test = null;
+    });
+
+    it('should add show class when show is called', () => {
+      const component = $$(test.cmp.element);
+      expect(component.hasClass(ResultActionsMenu.SHOW_CLASS)).toBe(false);
+
+      test.cmp.show();
+
+      expect(component.hasClass(ResultActionsMenu.SHOW_CLASS)).toBe(true);
+    });
+
+    it('should remove show class when hide is called', () => {
```

Comment:
I find the test on line 36 more useful and removes the need for this one which is more fragile. My perspective is when I click, I expect a certain behaviour. How that behaviour is implemented is less important. Should someone rename the methods `show` and `hide` (unlikely but to drive the point home), this test would break but it would not be telling me anything useful. The test should serve me, I should not serve the test :)